### PR TITLE
Propagate sandbox to massa-proof-of-stake-exports

### PR DESCRIPTION
**Pull request by adrien-zinger**
Very small fix :)

### CODE REVIEW

File: massa-proof-of-stake-exports/Cargo.toml [hunk 1]
```diff
@@ -28,3 +28,4 @@ massa_serialization = { path = "../massa-serialization" }
 # for more information on what are the following features used for, see the cargo.toml at workspace level
 [features]
 instrument = ["tokio/tracing", "massa_models/instrument"]
+sandbox = []
```
Comment (AurelienFT) [inline massa-proof-of-stake-exports/Cargo.toml:31]: Need to propagate here also.
```suggestion
sandbox = ["massa_models/sandbox"]
```

File: massa-bootstrap/Cargo.toml [hunk 2]
```diff
@@ -61,4 +61,8 @@ testing = [
     "massa_ledger_worker/testing",
     "massa_async_pool/testing",
 ]
-sandbox = ["massa_async_pool/sandbox", "massa_final_state/sandbox"]
+sandbox = [
+    "massa_async_pool/sandbox",
+    "massa_final_state/sandbox",
+    "massa_proof_of_stake_exports/sandbox",
```
Comment (AurelienFT) [inline massa-bootstrap/Cargo.toml:67]: Need to add all dependency that use sandbox such as massa_models

**Review by AurelienFT — COMMENTED** [review]
Also add on dependencies of massa-node. There is a lot of crate in dependency  of massa-node that need it.

File: massa-proof-of-stake-exports/Cargo.toml (hunk 1 above)
Comment (adrien-zinger) [reply massa-proof-of-stake-exports/Cargo.toml]: done

File: massa-bootstrap/Cargo.toml (hunk 2 above)
Comment (adrien-zinger) [reply massa-bootstrap/Cargo.toml:69]: done

**Comment by AurelienFT** [comment]
In case you didn't see because the layout of GitHub isn't very good : https://github.com/massalabs/massa/pull/2708#pullrequestreview-1008974829

**Comment by adrien-zinger** [comment]
> In case you didn't see because the layout of GitHub isn't very good : [#2708 (review)](https://github.com/massalabs/massa/pull/2708#pullrequestreview-1008974829)

Done :wink: . Nevertheless, it's good to know that it's a bit useless since the feature is already activated by a foreign module. You can look at the [feature unification spec](https://doc.rust-lang.org/cargo/reference/features.html#feature-unification). It's a weird behavior but... better that the npm one :rofl: 

The problem was just that nobody was activating the feature `sandbox` of "massa_proof_of_stake_exports".

**Comment by AurelienFT** [comment]
> Done 😉 . Nevertheless, it's good to know that it's a bit useless since the feature is already activated by a foreign module. 

Yes but for example you build in the repository of massa-bootstrap the other crates are built as a dependency and then need to have the features passed otherwise they don't have it enable.

**Review by AurelienFT — APPROVED** [review]

**Comment by AurelienFT** [comment]
bors merge

**Comment by AurelienFT** [comment]
bors merge

**Pull request merged**